### PR TITLE
fix: putRawString emoji

### PR DESCRIPTION
**Pull request by coolme200**

### CODE REVIEW

File: test/byte.test.js [hunk 1]
```diff
@@ -486,6 +486,21 @@ describe('byte.test.js', function () {
       bytes.putRawString('');
       bytes.toString().should.equal('<ByteBuffer>');
     });
+
+    it('should put emoji', function () {
+      // utf8
+      var bytes = ByteBuffer.allocate(1);
+      var str = new Buffer('aGVsbG/ppoPlsLI=', 'base64').toString('utf8');
```
Comment (fengmk2) [inline test/byte.test.js]: image

File: test/byte.test.js [hunk 2]
```diff
@@ -486,6 +486,21 @@ describe('byte.test.js', function () {
...
+      bytes.putRawString(str);
+      bytes.toString().should.eql('<ByteBuffer 68 65 6c 6c 6f e9 a6 83 e5 b0 b2>');
+      bytes.getRawString(0, 11).should.eql(str);
+      // gbk
+      var bytes = ByteBuffer.allocate(1);
+      var str = new Buffer('aGVsbG/wn4y8', 'base64').toString('utf8');
```
Comment (fengmk2) [inline test/byte.test.js]: image

**Comment by dead-horse** [comment]
屌！

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
byte@1.1.3

**Pull request merged**